package com.back.shared.member.out;

import org.springframework.stereotype.Service;
import org.springframework.web.client.RestClient;

@Service
public class MemberApiClient {
	private final RestClient restClient = RestClient.builder()
            .baseUrl("http://localhost:8080/api/v1/member")
            .build();

	public String getRandomSecureTip() {
		return restClient.get()
			.uri("/members/randomSecureTip")
			.retrieve()
			.body(String.class);
	}
}
